RetailCoder.VBE/Inspections/QuickFixes/AssignedByValParameterMakeLocalCopyQuickFix.cs

```diff
@@ -100,10 +100,18 @@ private void ReplaceAssignedByValParameterReferences(IModuleRewriter rewriter, s
         private void InsertLocalVariableDeclarationAndAssignment(IModuleRewriter rewriter, string localIdentifier)
         {
             var content = Tokens.Dim + " " + localIdentifier + " " + Tokens.As + " " + _target.AsTypeName + Environment.NewLine
-                + (_target.AsTypeDeclaration is ClassModuleDeclaration ? Tokens.Set + " " : string.Empty)
+                + (IsCtLExprContext(_target) ? Tokens.Set + " " : string.Empty)
                 + localIdentifier + " = " + _target.IdentifierName;
 
             rewriter.InsertBefore(((ParserRuleContext)_target.Context.Parent).Stop.TokenIndex + 1, "\r\n" + content);
         }
+
+        private bool IsCtLExprContext(Declaration target)
+        {
+            var argContext = target.Context as VBAParser.ArgContext;
+            var typeCtxt = argContext.asTypeClause().type().complexType();
```

Comment:
Wouldn't this throw a NRE if `argContext` has no `asTypeClause`?

    var typeCtxt = argContext.asTypeClause()?.type()?.complexType();